README.md

```diff
@@ -118,6 +118,21 @@ The normal Serverless deploy procedure will automatically bundle with Webpack:
 - Install Serverless Webpack as above
 - Deploy with `serverless deploy`
 
+### Use with serverless-offline
```

Comment:
@hassankhan Just saw, that we have 2 sections for serverless-offline. This one and one under "Simulate API Gateway locally". I will combine the two into this one (as we plan to remove serve) and commit that. Then we can just remove the whole "Simulate ..." section together with the serve implementation later.